tests/templates/kuttl/smoke/helm-bitnami-minio-values.yaml.j2

```diff
@@ -53,8 +53,6 @@ auth:
 service:
   type: NodePort
 
-{% if test_scenario['values']['s3-use-tls'] == 'true' %}
```

Comment:
@sbernauer I have removed the non-TLS S3 tests, even though they are still valid for older supported versions.

I wasn't sure how to make it selective through test-definitions.yaml (I can do it here in the template, but then it is less obvious).

Perhaps I can add the condition to the template, and rename the dimension to mention old versions only?

Or is it ok that we just stop testing non-TLS (I'd prefer that).

Reply:
> Or is it ok that we just stop testing non-TLS (I'd prefer that).

Works for me, is the easiest solution

Reply:
Accepted :smile: